src/apps/loyalty/server/template.tsx

```diff
@@ -3,13 +3,20 @@ interface TemplateData {
   html: string
   entrypoint: string
   bootstrapData?: string
-  sharify?: string
+  sharify?: any
   baseURL: string
 }
 
 export default ({styles, html, entrypoint, bootstrapData, sharify, baseURL}: TemplateData) => {
   const fontsURL = "//fast.fonts.net/cssapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.css"
-
```

Comment:
we should use `<script type="text/javascript" async="" src="https://cdn.segment.com/analytics.js/v1/FtkEwggd3GeIHHtE5qtHPfkY6upwDNHf/analytics.min.js"></script>` to use the js file already cached by the browser on artsy.net.

Then just call `analytics.load()` in `browser.ts`